verl/utils/megatron_utils.py

```diff
@@ -748,3 +755,97 @@ def tensor_generator():
         converted_names, converted_params = weight_converter.convert_param(cur_name, infer_params)
 
         yield from zip(converted_names, converted_params)
+
+
+def get_transformer_layer_offset(pipeline_rank, vp_rank, config: TransformerConfig):
```

Comment:
can we have a high-level description of the differences with megatron's original function?